src/retrace/retrace.py

```diff
@@ -2050,8 +2053,17 @@ def get_dump_level(self, task: RetraceTask) -> Optional[int]:
         cmd = ["makedumpfile", "-D", "--dump-dmesg", str(self._vmcore_path), str(dmesg_path)]
 
         result = None
-        lines = run(cmd, stdout=PIPE, stderr=DEVNULL,
-                    encoding='utf-8', check=False).stdout.splitlines()
+        t = CONFIG["ProcessCommunicateTimeout"]
+        log_info("Executing makedumpfile to obtain dump level")
+        try:
+            lines = run(cmd, stdout=PIPE, stderr=DEVNULL,
+                        encoding='utf-8', timeout=t,
```

Comment:
```suggestion
                        encoding="utf-8", timeout=t,
```

Reply:
I'm not opposed to this, but I wonder if this should go in a separate patch to clean all these up?  It looks like there's a lot of inconsistency with either 'utf-8' or "utf-8" in the code.  Do you want me to submit a separate cleanup patch for that?

Reply:
I can put a second patch on this series to clean this up.